x/ethbridge/handler.go

```diff
@@ -56,28 +39,30 @@ func NewHandler(
 
 // Handle a message to create a bridge claim
 func handleMsgCreateEthBridgeClaim(
-	ctx sdk.Context, cdc *codec.Codec, bridgeKeeper Keeper, msg MsgCreateEthBridgeClaim, sugaredLogger *zap.SugaredLogger,
+	ctx sdk.Context, cdc *codec.Codec, bridgeKeeper Keeper, msg MsgCreateEthBridgeClaim,
 ) (*sdk.Result, error) {
+
+	logger := bridgeKeeper.Logger(ctx)
 	var mutex = &sync.RWMutex{}
 	mutex.Lock()
 	defer mutex.Unlock()
 
-	status, err := bridgeKeeper.ProcessClaim(ctx, types.EthBridgeClaim(msg), sugaredLogger)
+	status, err := bridgeKeeper.ProcessClaim(ctx, types.EthBridgeClaim(msg))
 	if err != nil {
-		sugaredLogger.Errorw("bridge keeper failed to process claim.",
-			errorMessageKey, err.Error())
+		logger.Error("bridge keeper failed to process claim.",
+			"error:", err.Error())
 		return nil, err
 	}
 	if status.Text == oracle.SuccessStatusText {
-		if err = bridgeKeeper.ProcessSuccessfulClaim(ctx, status.FinalClaim, sugaredLogger); err != nil {
-			sugaredLogger.Errorw("bridge keeper failed to process successful claim.",
-				errorMessageKey, err.Error())
+		if err = bridgeKeeper.ProcessSuccessfulClaim(ctx, status.FinalClaim); err != nil {
+			logger.Error("bridge keeper failed to process successful claim.",
+				"error:", err.Error())
```

Comment:
Why did you switch this to using a string instead of errorMessageKey?

Reply:
I can revert. I found it confusing this was a key, but makes sense for your logging system.

Reply:
reverted